docker/sui-antithesis/docker-compose-antithesis.yaml

```diff
@@ -123,7 +123,7 @@ services:
     environment:
       - ENABLE_WRITEBACK_CACHE=1
       - RUST_BACKTRACE=1
-      - RUST_LOG=info,jsonrpsee=error
+      - RUST_LOG=info,sui_core=debug,sui_node=debug,jsonrpsee=error
```

Comment:
consensus?

Reply:
Consensus debug output is very noisy, and afaik consensus team has not been relying on antithesis much lately. We can always re-enable it later. At `info` level we still get a log line each time a commit is decided.

Reply:
When debugging consensus issues, per txn log is also not very useful. Why filtering is not helpful here?

Reply:
And I don't think consensus logging is noisy. At DEBUG level, they are basically logging the lifecycle of blocks, which may seem more verbose when txn tps is lower than block / sec in tests. It is true that so far there is no consensus bug that only showed up in antithesis. We can definitely reduce the consensus logging level if it really helps.

Reply:
it is noisy - here are the top modules in a recent log i downloaded from an antithesis run: 

```
; grep -Eo '[a-z_]+::[:a-z_]+:' ./config/logs_dump/logs/validator2/full_logs.log | sort | uniq -c | sort -r
3163 consensus_core::universal_committer:
2379 sui_core::checkpoints:
1279 consensus_core::block_manager:
1255 consensus_core::core:
1077 consensus_core::authority_service:
1007 consensus_core::dag_state:
 940 sui_core::authority::authority_per_epoch_store:
 648 sui_core::authority:
 612 sui_network::state_sync:
 596 sui_core::checkpoints::checkpoint_executor:
 414 sui_core::consensus_adapter:
 402 consensus_core::base_committer:
 ```

(After this PR `sui_core::checkpoints` will be much less noisy as well.)

Reply:
As discussed, the consensus logs are useful. They have comparably more entries because TPS in tests are low or zero. I have asked to turn up simulated latency in the test system.